### DISCUSSION TITLE
Airflow 1 or 2 BigQuery Profiling (Memory Scan/Cost) per Dag

### DISCUSSION THREAD

**Discussion by MohammedDiab** [apache/airflow#28112] [Q&A]
Dears, 

I want to know how much memory scan per Dag that utilises BigQueryOperators, I tried to search for such feature but I haven't find so far, it is something that my team is willing to contribute if it doesn't exist

**Comment by potiuk** [comment]
Not sure if this should be part of the operators  - I think it rather should be an external tool (not even part of the Airflow Google Provider) that uses Airlfow's REST API to get appropriate information (BQ Job ids) and generate some statistics out of that.

You could develop and open-source it completely outside of Apache Airflow and make it part of the ecosystem of Airflow: https://airflow.apache.org/ecosystem/
